velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -133,6 +138,37 @@ TEST_F(RemainderTest, int64) {
   EXPECT_EQ(-1, remainder<int64_t>(INT64_MIN, INT64_MAX));
 }
 
+TEST_F(RemainderTest, double) {
+  EXPECT_DOUBLE_EQ(0.0, remainder_value<double>(2.0, 1.0));
+  EXPECT_DOUBLE_EQ(1.0, remainder_value<double>(5.0, 2.0));
+  EXPECT_DOUBLE_EQ(-1.0, remainder_value<double>(-5.0, 2.0));
+  EXPECT_DOUBLE_EQ(0.5, remainder_value<double>(1.5, 1.0));
+  EXPECT_DOUBLE_EQ(0.0, remainder_value<double>(0.0, 1.0));
+  EXPECT_DOUBLE_EQ(2.0, remainder_value<double>(2.0, std::numeric_limits<double>::infinity()));
+
+  EXPECT_TRUE(std::isnan(remainder_value<double>(std::numeric_limits<double>::quiet_NaN(), 1.0)));
```

Comment:
We can define it as a variable and reuse in the test to improve readability. Ditto for the others.

Reply:
done~